keyboards/stenokeyboards/polyglot/keymaps/everything_qwerty/info.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Keymap level `info.json` files are invalid, and should be removed.

(This would usually be flagged by the lint check but for some reason its not working)

Reply:
how would I implement rgb then?

Reply:
Why would it be required to be at the keymap level? Implement it at the keyboard level?
Otherwise use the config.h and rules.mk options.

As a note: It doesnt function so the submission hasnt been tested.

Reply:
added suggestion in `config.h` which should allow this to be removed 👍

Reply:
@petercpark 
following on from both @lesshonor and @zvecr comments - does this PCB have RGB lighting (not to be confused with RGB matrix) built into the PCB?